src/backend/gl/src/command.rs

```diff
@@ -790,5 +842,26 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
     }
 }
 
+fn push_cmd_internal(id: &u64, memory: &mut Arc<Mutex<pool::BufferMemory>>, buffer: &mut BufferSlice, cmd: Command) {
+    let slice = {
```

Comment:
is this scope needed here?

Reply:
I just moved the body from `push_cmd`. I agree though, doesn't appear to be necessary so I'll remove it